src/Parsers/ParserSystemQuery.cpp

```diff
@@ -245,6 +245,14 @@ bool ParserSystemQuery::parseImpl(IParser::Pos & pos, ASTPtr & node, Expected &
             break;
         }
 
+        case Type::SYNC_DATABASE:
+        {
+            parseQueryWithOnCluster(res, pos, expected);
```

Comment:
Thought about implementation with `ON CLUSTER`, not sure how it should work

Reply:
It should work without additional changes, let's try to add a query like
`system sync database test_sync_database on cluster test_sync_database`
to the test.

Reply:
But after executing this query we can't guarantee that all database replicas are up-to-date. Because we only wait for `current_task` to be equal to the `dummy` that we created